EG23.tex

```diff
@@ -157,20 +161,32 @@ \subsection{Reactors}
 \begin{table}[h!]
     \centering
     \begin{tabularx}{350pt}{lXXX}
-    \hline
-    Core Properties       &	LWR     &	SFR 1   &	SFR 2     \\
-    \hline
-    Rated Power, [MWe]    &	1000		&	400     &	400       \\
-    Thermal Efficiency    &	N.A.$^1$	&	0.4     &	0.4       \\
-    Capacity Factor       &	0.90		&	0.90		&	0.90      \\
-    Number of batches     &	6       &	5       &	3         \\
-    Cycle length, [month] &	9       &	13      &	18        \\
-    Core Inventory, [tHM] &	6$\times$14.784 & 5$\times$7.524 & 3$\times$11.466  \\
-    \hline
+      \hline
+      Core Specifications   &  LWR             &  SFR 1           &  SFR 2            \\
+      \hline
+      Rated Power, [MWe]    &  1000            &  400             &  400              \\
+      Thermal Efficiency    &  0.33            &  0.4             &  0.4              \\
+      Capacity Factor       &  0.90            &  0.90            &  0.90             \\
+      Burnup, [Gwd/tHM]     &  50              &  47.501          &  42.397           \\
+      Number of batches     &  3               &  5.44            &  4.44             \\
+      Cycle length, [month] &  18              &  12              &  12               \\
+      Core Inventory, [tHM] &  3$\times$29.565 &  37.62           &  34.40            \\
+                                                                                      \\
+      \multicolumn{4}{l}{Cyclus inplementation}                                       \\ 
+      \hline
+      \hline
+      Assembly size [kgHM]  &  14784           &  7524            &  11466            \\
+      Assembly in the core  &  6               &  5               &  3                \\
+      Assembly per batches  &  1               &  1               &  1                \\
+      Cycle length, [month] &  9               &  13              &  18               \\
+      Core Inventory, [tHM] &  6$\times$14.784 &  5$\times$7.524  &  3$\times$11.466  \\
+%% I have increase the size of the batch by 1 to avoid a bid bug I
+%% experience... which we have corrected after... But I did not revert the natch modification 
+      \hline
```

Comment:
@gonuke I have rethink the reactor table, with 2 part : 
- the specification
- the implementation/information require by cyclus

I'll reformat all table that way...